CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- Changed the element of `IconMenu` to our `Box` component ([@InstaK](<(https://github.com/Kemosabert)>) in [#324](https://github.com/teamleadercrm/ui/pull/324))
```

Comment:
You forgot to change the link 😄

Reply:
I also don't know why you added the extra `<( ... )>`

Reply:
what 😄 

 :ok: by @InstaK on 30/07/2018 14:45:48